spec/Staking.md

```diff
@@ -47,16 +47,40 @@ makeKey(...fragments) = rlp([
     A `delegatee` is an `AccountId`, and the `quantity` is a non-zero `u64` amount of CCS that a `delegator` delegated to a `delegatee`.
     The RLP-encoded non-empty list should be sorted by a `delegatee` in ascending order, and every `delegatee` should be unique.
 
-## Pending Revocations
+## List of Candidates
 
-  * State Key: `makeKey("Revocations")`
+  * State Key: `makeKey("Candidates")`
+  * Value: `rlp(list of [pubkey, deposit, nominations_ends_at, metadata])`
 
-  * Value: `rlp(list of [delegator, delegatee, endTime, quantity])`
+    The `pubkey` is a public key of a self-nominated candidate. The `deposit` is a `u64` amount of CCS deposited by the candidate and `nomination_ends_at` is a `u64` term id when the entry is expired. The `metadata` is a `bytes` that can store a short amount of data that express or advertise themselves. The order of it is constantly changed as the candidates send self-nominate transaction and the term finishes. See the 'Candidate prioritizing' section of [dynamic validator](./Dynamic-Validator.md#Candidate-prioritizing).
```

Comment:
The `pubkey` is a public key of a self-nominated candidate. The `deposit` is a `u64` amount of CCS deposited by the candidate and `nomination_ends_at` is a `u64` term id when the entry will expire. The `metadata` is a `bytes` that can store a short amount of data that expresses or advertises themselves. The order of it is constantly changed as the candidates send a self-nominate transaction and the term finishes. See the 'Candidate prioritizing' section of [dynamic validator](./Dynamic-Validator.md#Candidate-prioritizing).